src/shogun/mathematics/linalg/ratapprox/logdet/opfunc/DenseMatrixExactLog.cpp

```diff
@@ -15,27 +15,22 @@
 
 #include <shogun/lib/SGVector.h>
 #include <shogun/lib/SGMatrix.h>
-#include <shogun/lib/computation/aggregator/StoreScalarAggregator.h>
-#include <shogun/lib/computation/engine/IndependentComputationEngine.h>
 #include <shogun/mathematics/linalg/linop/DenseMatrixOperator.h>
 #include <shogun/mathematics/linalg/ratapprox/logdet/opfunc/DenseMatrixExactLog.h>
-#include <shogun/mathematics/linalg/ratapprox/logdet/computation/job/DenseExactLogJob.h>
 
 using namespace Eigen;
 
 namespace shogun
 {
 
-CDenseMatrixExactLog::CDenseMatrixExactLog()
-	: COperatorFunction<float64_t>(NULL, NULL, OF_LOG)
-{
-	SG_GCDEBUG("%s created (%p)\n", this->get_name(), this)
+	CDenseMatrixExactLog::CDenseMatrixExactLog()
```

Comment:
you have a lot of whitespace changes not sure whether that is due to the auto-formatter.
In any case, search for "style" in the first travis job output to get the comment how to fix formatting

Reply:
This was a style issue I have fixed it now... i think